patterns/centered-statement.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Title: Centered Statement
+ * Slug: twentytwentyfour/centered-statement
+ * Categories: text
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"1.3","fontStyle":"normal","fontWeight":"400"},"spacing":{"padding":{"top":"12rem","bottom":"12rem","right":"13rem","left":"13rem"}}},"backgroundColor":"base","textColor":"contrast","fontSize":"large","fontFamily":"cardo"} -->
```

Comment:
Here, we shouldn't duplicate the values that are the defaults, like the font style, font weight, background color, and text color.
We also don't need to duplicate the settings and add them to both the group and the paragraph.

Reply:
Instead of hardcoding 12  and 13 rem,  please select a spacing preset.